### ISSUE TITLE
Silent mode and not open a new browser on start

### ISSUE THREAD

**Issue by Foxandxss** [johnpapa/lite-server#84]
For angular.io we use `http-server` to run e2e tests, but for certain reasons we would need to swap to `lite-server` to run the e2e. The default verbose mode and the fact that opens a new browser when it runs are problematic for this tho.

I talked with @johnpapa and he agreed that two options to make the output completely silent and and an option to disable the auto opening of the browser are nice to have.

I would really really appreciate if one of the team members in here that knows how this server work make this options available. I have no knowledge about lite-server or browser sync and I discovered that we need to use this for e2e just in the middle of a very big PR :/

Thanks!

**Comment by Foxandxss** [comment]
Oh, seems like I am not in a hurry anymore. Still nice to have options, will peek if no one is willing to.

**Comment by johnpapa** [comment]
These settings should do it ... may not need a PR at all. Let us know

```
  open: false
  logLevel: "silent",
  server: {
    middleware: {
      0: null
    }
  }
```

**Comment by Foxandxss** [comment]
That works nice John. Would be lovely to remove this as well:

```
[1] ** browser-sync config **
[1] { injectChanges: false,
[1]   files: [ './**/*.{html,htm,css,js}' ],
[1]   watchOptions: { ignored: 'node_modules' },
[1]   server: { baseDir: './', middleware: [ [Function] ] },
[1]   open: false,
[1]   logLevel: 'silent' }
[0] 13:21:02 - Compilation complete. Watching for file changes.
```

But muuch better than before.

Since my PR is on hold and we are not sure what is going to come. Let's close this and speak later if needed.

**Comment by johnpapa** [comment]
try the `silent` branch

**Comment by Foxandxss** [comment]
No luck, outputs the same thing.

**Comment by johnpapa** [comment]
ah, sorry ... still waking up. try again

**Comment by Foxandxss** [comment]
Purrfect.

**Comment by johnpapa** [comment]
closed with #85
